docs/notebooks/Common_Gotchas_in_JAX.ipynb

```diff
@@ -776,7 +776,7 @@
     "id": "m0XZLP7nuYdE"
    },
    "source": [
-    "The output is what we would expect, but this hides potential performance issues under the hood. In JAX's tracing and JIT compilation model, each element in a Python list or tuple is treated as a separate JAX variable, and individually processed and pushed to device. This can be seen in the jaxpr for the ``permissive_sum`` function above:"
+    "The output is what we would expect, but this hides potential performance issues under the hood. In JAX's tracing and JIT compilation model, each element in a Python list or tuple is treated as a separate JAX variable, and individually processed and pushed to the device. This can be seen in the jaxpr for the ``permissive_sum`` function above:"
```

Comment:
I think this reads better as originally written. "pushed to device" is a common idiom when talking about data device location.

Reply:
Reverted this change.